src/zoid/buttons/component.jsx

```diff
@@ -719,6 +719,18 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
         },
       },
 
+      referrer_domain: {
+        type: "string",
+        required: false,
+        value: () => {
+          const referrer = window.document.referrer;
+          const matches = referrer.match(/:\/\/([^/?]+)/);
+          if (matches) {
+            return matches[1];
+          }
+        },
```

Comment:
A few notes about this implementation:

1. using `URL()` failed the eslint check because ie9 does not support `URL`
2. There are more comprehensive regexes we could use (https://stackoverflow.com/a/26987741) but I thought the simpler regex would be more readable and we actually don't _need_ a valid domain, just an identifier that is domain-like.
3. I thought about moving this to a shared util location but:
  - [belter](https://github.com/search?q=repo%3Akrakenjs%2Fbelter%20domain&type=code) doesn't have any domain-related utils
  - [cross-domain-utils](https://github.com/krakenjs/cross-domain-utils/blob/main/src/utils.js#L91) is coupled to `window.location` (not `document.referrer`)
  - [paypal-checkout-components](https://github.com/paypal/paypal-checkout-components/tree/main/src/lib) does not have any related URL parsing

I'm definitely open to moving it! wdyt?

Reply:
We don't support IE11 and earlier, you should always feel empowered to us a feature as long as its supported in most modern browsers